.github/workflows/ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dsherret/rust-toolchain-file@v1
       - name: Cache cargo
-        uses: Swatinem/rust-cache@v2
+        uses: Swatinem/rust-cache@v4
```

Comment:
This is wrong, and should be reverted.

Reply:
Thanks, fixed! [`508e049` (#88)](https://github.com/dprint/dprint-plugin-prettier/pull/88/commits/508e0495d10be1bec1d768bcd1f86ef7d79deb53)